src/services/openWeatherService.ts

```diff
@@ -105,19 +89,24 @@ export default class OpenWeatherMap {
     } catch (error) {
       output = error;
     }
-    await countapi.hit(process.env.VUE_APP_COUNTER_NAME ?? 'WeatherApp', 'current-weather')
-      .then((result: { value: number; }) => { this.callCountCurrent = result.value; });
 
     return output;
-  }
+  },
+
+  async searchCoordsByCity(query: string): Promise<GeoDirectResponse[]> {
+    const url = `http://api.openweathermap.org/geo/1.0/direct?&q=${query}&appId=${store.state.apiKeys[OPEN_WEATHER]!}`;
+    const response = await fetch(url);
+    // TODO add call to count API, but should we track different endpoint calls separately?
```

Comment:
My original thought was to track per endpoint at OpenWeatherMap. Whether we called the get Forecast with coordinates or with the city, OpenWeatherMap considered it one API and both would count toward the same Monthly total.

Reply:
I kind of punted on this. The total call count is all that matters, and even that doesn't *really* matter in the sense that we'll never approach millions of calls. Still, it's fun to see and it would also be fun to see the calls broken down by endpoint. We'd just have to calculate the total if we don't track it separately (could use a Vuex [getter](https://vuex.vuejs.org/guide/getters.html)), and I guess we'd want the count to reset each month. In any case I figured we can just deal with it separately in #189.